lib/author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_accessor :first_name, :last_name
+
+  def initialize(first_name = 'Undefined', last_name = 'Undefined')
+    @last_name = last_name
+    @first_name = first_name
+    @items = items
```

Comment:
[Required] @item should be an array `@items = items` the variable items is not in the initialized variable. Kindly make `@items = []`

Reply:
@darikmohammed thanks for the review, I will fix it asap.